app/assets/javascripts/wizard.js

```diff
@@ -312,4 +312,19 @@
   };
 
   $(".js-wizard").refusalWizard({ debug: false });
+
+
+  $('[data-block="exemption"] fieldset').after("<button class='button-secondary maximise-questions'>Show more</button>");
```

Comment:
Any user-visible text data-attributes on the HTML so that we can translate them. Should be existing patterns to follow for this.

Reply:
Ah, yep, makes sense. I've had a browse around the codebase but can't find a pattern to follow, would you mind pointing me towards one? Cheers

Reply:
Ah, sorry, this was no where as obvious as I thought it was. Here are a couple:

* https://github.com/mysociety/alaveteli/blob/3add3721d0dc2d2011d63127c2f51fc637c5e34d/app/views/request/_after_actions.html.erb#L89
* https://github.com/mysociety/alaveteli/blob/3add3721d0dc2d2011d63127c2f51fc637c5e34d/app/views/alaveteli_pro/batch_request_authority_searches/index.html.erb#L51-L54
* https://github.com/mysociety/alaveteli/blob/3add3721d0dc2d2011d63127c2f51fc637c5e34d/app/views/alaveteli_pro/info_requests/_select_authority_form.html.erb#L13